dumpling/export/status.go

```diff
@@ -48,6 +48,25 @@ func (d *Dumper) runLogProgress(tctx *tcontext.Context) {
 	}
 }
 
+type Midparams struct {
+	TotalTables       float64
+	CompletedTables   float64
+	FinishedBytes     float64
+	FinishedRows      float64
+	EstimateTotalRows float64
+}
+
+func (d *Dumper) GetParameters() (midparams *Midparams) {
```

Comment:
Can you use this function to rewrite `status.go: L32~L35`? They look almost the same.

Reply:
Thanks, I will try to rewrite this lines later.